### PR TITLE
Default Hitmarker

### PR DESCRIPTION
**Pull request by ZerGo0**
I could make it so that it fades out, but I'm too lazy right now.

Donated btw. I forgot to add my name to the message. ;)

### CODE REVIEW

File: Osiris/Hacks/Visuals.cpp [hunk 1]
```diff
@@ -307,3 +308,41 @@ void Visuals::hitEffect(GameEvent* event) noexcept
         }
     }
 }
+
+void Visuals::hitMarker(GameEvent* event) noexcept
+{
+    const auto localPlayer = interfaces.entityList->getEntity(interfaces.engine->getLocalPlayer());
+
+    if (!localPlayer || !localPlayer->isAlive() || !config.visuals.hitMarker || config.visuals.hitMarker == 0)
```
Comment (ClaudiuHKS) [inline Osiris/Hacks/Visuals.cpp]: `if (!localPlayer || !localPlayer->isAlive() || !config.visuals.hitMarker || config.visuals.hitMarker == 0)`

→

`if (!localPlayer || !localPlayer->isAlive() || config.visuals.hitMarker == 0)`

Comment (ZerGo0) [reply Osiris/Hacks/Visuals.cpp]: 🤫 fixed

**Comment by notgoodusename** [comment]
instead of switch do if(int != 0)

**Comment by ZerGo0** [comment]
People might want to add more variations in the future, that's why I coded it like that.

**Comment by danielkrupinski** [comment]
Thanks for your contribution! Donation appreciated 😊

**Pull request merged**